### PR TITLE
Rst formatting issue fix #42

### LINKED ISSUE #42: The Readme.rst formatting for code not showing correctly
The code block in the README file is not showing correctly, on check .rst format there were issues.
System Message: WARNING/2 (e:\bkp\linux\code\python\fetchcode\README.rst, line 15)

The code block shows
` from fetchcode import fetch url = 'A Http or FTP URL' location = 'Location of file' # This returns a response object which has attributes # 'content_type' content type of the file # 'location' the absolute location of the files that was fetched # 'scheme' scheme of the URL # 'size' size of the retrieved content in bytes # 'url' fetched URL resp = fetch(url = url) `

### PR DESCRIPTION
**Pull request by bobquest33**
Code format issues in rst has been fixed

### CODE REVIEW

**Comment by TG1999** [comment]
Hi, thanks for this. Can you remove the line number 3 of Python code block, the location one :)

**Comment by bobquest33** [comment]
Can you please share the exact code which you are highlighting, or add a comment in code changes part

File: README.rst [hunk 1]
```diff
@@ -12,22 +12,23 @@ Then install all the requirements using
...
-# 'size' size of the retrieved content in bytes
-# 'url' fetched URL
-resp = fetch(url = url)
-```
+
+.. code-block:: python
+
+    from fetchcode import fetch
+    url = 'A Http or FTP URL'
+    location = 'Location of file'
```
Comment (TG1999) [inline README.rst:27]: Line number 27 here :)

Comment (bobquest33) [reply README.rst]: Fixed

Comment (TG1999) [reply README.rst]: Thanks, can you squash your commits :)

Comment (bobquest33) [reply README.rst]: Done, is now is it ok

Comment (bobquest33) [reply README.rst]: Does it have good quality to merge the code

File: README.rst [hunk 2]
```diff
@@ -12,22 +12,22 @@ Then install all the requirements using
 `pip3 install -r requirements.txt`
 
 Running test suite
-#################
+##################
 
 To run test suite
 `python3 -m pytest`
```
Comment (TG1999) [inline README.rst:18]: can you also make this line indented :) ?

File: README.rst [hunk 3]
```diff
@@ -12,22 +12,22 @@ Then install all the requirements using
 `pip3 install -r requirements.txt`
```
Comment (TG1999) [inline README.rst:12]: can you also make this line indented :) ?

File: README.rst [hunk 4]
```diff
@@ -1,6 +1,6 @@
-*****
+*********
 Fetchcode
-*****
+*********
 It is a library to reliably fetch code via HTTP, FTP and version control systems. 
 
 Installation
```
Comment (TG1999) [inline README.rst:6]: can you also make this line indented :) ? Line number 9 in Installation

**Comment by TG1999** [comment]
Ping @bobquest33

**Comment by TG1999** [comment]
@pombredanne can you please have a look and merge this :)

**Comment by pombredanne** [comment]
@bobquest33 thanks!
Why do you remove test code though? that's completely unrelated to documentation.

**Comment by pombredanne** [comment]
If something needs to be deleted, this should be in another PR IMHO and discussed there

**Comment by TG1999** [comment]
Hi @pombredanne , I removed that, I told you earlier that we have some redundant code and some redundant tests were added for them, so removed them

**Comment by pombredanne** [comment]
@TG1999 OK fair enough!
LGTM then :+1:

**Pull request merged**